components/Slider/src/MDCSlider.m

```diff
@@ -583,14 +587,23 @@ - (void)thumbTrackValueChanged:(__unused MDCThumbTrack *)thumbTrack {
   [self sendActionsForControlEvents:UIControlEventValueChanged];
   UIAccessibilityPostNotification(UIAccessibilityAnnouncementNotification, self.accessibilityValue);
   if (@available(iOS 10.0, *)) {
-    if (self.hapticsEnabled) {
+    if (self.hapticsEnabled && (_firstValueChange || _previousValue != _thumbTrack.value)) {
+      BOOL crossesAnchor =
```

Comment:
could this possibly be simplified to multiple BOOLs that explains more easily for the reader each part of the conditionals?

example:
BOOL crossesAnchorToTheLeft
BOOL crossesAnchorToTheRight
etc